### ISSUE TITLE
What loss function do you use?

### ISSUE THREAD

**Issue by jjeamin** [Sxela/ArcaneGAN#8]
Hello, thank you for your wonderful work. 

What loss function do you use?

+ How to learn to improve performance?

**Comment by Sxela** [comment]
Hello and thank you!
An old variation of LPIPS + gram + L1

**Comment by jjeamin** [comment]
Did you learn only the Unet generator without using the discriminator? (no GAN Loss)

**Comment by Sxela** [comment]
Ah, sorry, there is a discriminator and adversarial loss, too, after a short warmup.

**Comment by jjeamin** [comment]
Can I know the training settings? (epoch, loss weights, etc.)

**Comment by Sxela** [comment]
They differ from exact dataset to dataset, but you can get the overall idea from this notebook for paired image training:  https://www.patreon.com/posts/training-model-60280567

**Comment by jjeamin** [comment]
Many Thanks!!
